examples/spec/integration/describe_namespace_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'temporal/errors'
+
+describe 'Temporal.describe_namespace' do
+  it 'returns a value' do
+    description = 'Namespace for temporal-ruby integration test'
+    begin
+      Temporal.register_namespace('a_test_namespace', description)
+    rescue Temporal::NamespaceAlreadyExistsFailure
+    end
+    result = Temporal.describe_namespace('a_test_namespace')
```

Comment:
Does this work every time without flakes? I had to `sleep 15` during the integration spec setup to allow it to settle

Reply:
I have never seen it flake and we've been running it for a couple months.